packages/approval-controller/src/ApprovalController.ts

```diff
@@ -203,8 +203,8 @@ export type ApprovalControllerMessenger = RestrictedControllerMessenger<
   typeof controllerName,
   ApprovalControllerActions,
   ApprovalControllerEvents,
-  never,
-  never
+  string,
+  string
```

Comment:
@mcmire helped me with getting the messenger working in the new NFTController test added here, and we found that this change was required for typescript to be [happy with this line](https://github.com/MetaMask/core/pull/1415/files#diff-22f99da950cfe9d51506651602f04e50afcd3805291cbfa9e18165dce3568f25R158)